linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/LinkerdAdmin.scala

```diff
@@ -53,6 +54,20 @@ object LinkerdAdmin {
     Handler("/logging.json", new LoggingApiHandler())
   )
 
+  def identifierHandler(lc: Linker.LinkerConfig, linker: Linker): Handler = {
+    linker.routers.head.protocol.configId
+
+    val identifiers = linker.routers.collect {
+      case router if router.protocol.configId == "http" =>
+        val RoutingFactory.DstPrefix(pfx) = router.params[RoutingFactory.DstPrefix]
+        val RoutingFactory.BaseDtab(baseDtab) = router.params[RoutingFactory.BaseDtab]
+        val Http.param.HttpIdentifier(id) = router.params[Http.param.HttpIdentifier]
+        router.label -> id(pfx, baseDtab)
```

Comment:
Keep in mind this will create a second identifier.  This is fine but we should be aware of this and try to make sure identifiers don't create expensive resources.